packages/edit-site/src/components/block-editor/index.js

```diff
@@ -39,13 +41,32 @@ export default function BlockEditor( { settings: _settings } ) {
 			},
 		};
 	}, [ canUserCreateMedia, _settings ] );
-	const [ blocks, setBlocks ] = useState( [] );
+	const [ content, _setContent ] = useEntityProp(
+		'postType',
+		'wp_template',
+		'content'
+	);
+	const initialBlocks = useMemo( () => {
+		if ( typeof content !== 'function' ) {
+			const parsedContent = parse( content );
+			return parsedContent.length ? parsedContent : undefined;
+		}
+	}, [] );
+	const [ blocks = initialBlocks, setBlocks ] = useEntityProp(
+		'postType',
+		'wp_template',
+		'blocks'
+	);
+	const setContent = useCallback( ( nextBlocks ) => {
+		setBlocks( nextBlocks );
+		_setContent( serialize( nextBlocks ) );
```

Comment:
Do we need to set content (run serialization) on evey change? Can't we do something like the editor (use a function edit)

Reply:
I'm replacing this with `useEntityBlockEditor` in the follow-up PR. Can we ignore it for now?